app/config/site.ts

```diff
@@ -0,0 +1,24 @@
+interface SiteConfig {
+	name: string;
+	title: string;
+	description: string;
+	localeDefault: string;
+	links: {
+		discord: string;
+		twitter: string;
+		github: string;
+	};
+}
+
+export const siteConfig: SiteConfig = {
+	name: "VoiceDeck",
+	title: "VoiceDeck - From Individual Actions to Collective Impact",
+	description:
+		"We enable journalists to effect real change by bringing critical stories to light. Your contributions directly support this mission, sustaining  journalism and bolstering investigative reporting that matters.",
```

Comment:
if you remove "Voicedeck - " from the title and change to sentence case, we can pull this in to `_index.tsx` for the hero content. also there is an extra space after "sustaining".
is it meant to be used somewhere else?

Reply:
Good catch.
I was thinking we could use the config to populate the metadata for the SEO also.

Reply:
done